contrib/packs/actions/pack_mgmt/get_pack_dependencies.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2019 Extreme Networks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import six
+
+from st2common.constants.pack import PACK_VERSION_SEPARATOR
+from st2common.content.utils import get_pack_base_path
+from st2common.runners.base_action import Action
+from st2common.util.pack import get_pack_metadata
+
+STACKSTORM_EXCHANGE_PACK = 'StackStorm-Exchange/stackstorm-'
```

Comment:
This is not desired assumption or mask.

What if user writes `stackstorm-EXCHANGE/stackstorm-`
What if user maintains their own pack index, which is possible via https://docs.stackstorm.com/packs.html#hosting-your-own-pack-index

Reply:
Will rework on parse pack input and without make any assumptions.